.jenkins/Jenkinsfile2

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env groovy
```

Comment:
This file should have a more descriptive name.  Do we want to move the other jenkinsfile into here as well?

Reply:
Yeah, definitely agree about the name. Not sure what to call it, definitely open to suggestions. As for moving the other jenkinsfile here, sure. I've added .jenkins/jenkins to RocmSoftwarePlatform as an acceptable Jenkins file location, so I can move the current Jenkins file.

Reply:
The main thing would be to put 'rocblas' somewhere in the name so that it's obvious that this is building that project.